packages/shared/src/components/cards/RaisedLabel.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import styles from './Card.module.css';
+import { SimpleTooltip } from '../tooltips/SimpleTooltip';
+
+export function RaisedLabel({
+  trending,
+  listMode,
+  type,
+}: {
+  trending: number;
+  listMode?: boolean;
+  type: 'hot' | 'pinned';
+}): ReactElement {
+  const description = `${trending} devs read it last hour`;
```

Comment:
This will make it very specific to trending though right?

Reply:
I agree, I think we can make the description come from the prop.